src/event/service.js

```diff
@@ -38,6 +38,33 @@ angular
     /* Cities
     /*==============================================================*/
 
+   /**
+    * List of events for the provided city. 
+    * Returns a promise. Supports additional 'options'.
+    *
+    * @method listEventsForCity
+    * @param {Object} options 
+    * @param {String} options.cityId - ID of a city
+    * @param {String} options.fields - CSV of fields to return
+    * @param {String} options.start  - unix start date for events
+    * @param {String} options.end    - unix end date for events
+    */
+
+    EventService.prototype.listEventsForCity = function(options) {
```

Comment:
Is anyone still using the `listForCity`? if they both do the same thing would be nice to only have 1.

I like the naming of `listForCity`, the `listEventsForCity` seem redundant since we are in the `EventService` so we know that we are getting a list of Events.